### DISCUSSION TITLE
Help!  "Connection to server storage failed"

### DISCUSSION THREAD

**Discussion by kymwkl** [YetiForceCompany/YetiForceCRM#17307] [How to]
Hello everyone.  I tried to configure roundcube and it's several hours and I'm hitting the same error no matter what I do.

Here are my credentials:
domain: semmel.com
imap and smtp host: mail.semmel.com

Firstly, I'm not sure I should use semmel.com or mail.semmel.com in the configuration so I tried both.  In the configuration, i tried these all these pairs but none of them works.  I'm on version YF 6.4.

tls://imap.semmel.com:993
tls://smtp.semmel.com:587

ssl://imap.semmel.com:993
ssl://smtp.semmel.com:587

tls://imap.mail.semmel.com:993
tls://smtp.mail.semmel.com:465

tls://imap.semmel.com:143
tls://smtp.semmel.com:465

tls://imap.mail.semmel.com:143
tls://smtp.mail.semmel.com:465

Please help.  Thank you.
Kym

**Comment by blu-IT** [comment]
Hi Kym,

when your mailservers hostname is semmel.com or mail.semmel.com you should not configure imap.mail.semmel.com etc.

So, the correct setting dependinmg on your mailserver configuration should be:

```
IMAP:
ssl://semmel.com:993 or
ssl://mail.semmel.com:993
or
tls://semmel.com:993 or
tls://mail.semmel.com:993

SMTP
ssl://semmel.com:587 or
ssl://mail.semmel.com:587
or
tls://semmel.com:587 or
tls://mail.semmel.com:587
or
ssl://semmel.com:465 or
ssl://mail.semmel.com:465
or
tls://semmel.com:465 or
tls://mail.semmel.com:465

```
